packages/block-editor/src/components/block-navigation/dropdown.js

```diff
@@ -53,7 +53,7 @@ function BlockNavigationDropdownToggle( { isEnabled, onToggle, isOpen } ) {
 	);
 }
 
-function BlockNavigationDropdown( { isDisabled } ) {
+function BlockNavigationDropdown( { isDisabled, withBlockNavigationSlots } ) {
```

Comment:
I think it'd be good to make this `__experimentalWithBlockNavigationSlots`, as it's a publicly exported prop, and it's hard to say what the future is, this may become the default at some point.